README.md

```diff
@@ -1,6 +1,6 @@
 [![Sous](https://circleci.com/gh/fourkitchens/sous-drupal-project.svg?style=svg)](https://app.circleci.com/github/fourkitchens/sous-drupal-project/pipelines)
 <br/>
-<img style="max-width: 400px;" src="https://github.com/fourkitchens/sous-drupal-distro/blob/master/themes/sous_admin/assets/images/sous.svg" alt="Sous featuring Emulsify">
+<img style="max-width: 400px;" src="https://github.com/fourkitchens/sous-drupal-distro/blob/4.x-beta/themes/sous_admin/assets/images/sous.svg" alt="Sous featuring Emulsify">
```

Comment:
@callinmullaney This should probably get updated after you merge this.  While it might continue to work we should probably point things to the default branch, or if you wanted to do 4.x we can probably do that branch minus the beta.

Reply:
Will do!

Reply:
Will be done later.